includes/class-sensei-course.php

```diff
@@ -2321,87 +2336,112 @@ public static function add_course_user_status_class( $classes, $class, $course_i
 	 * @param WP_Post $course
 	 */
 	public static function the_course_action_buttons( $course ) {
```

Comment:
Good cleanup! 👏

Reply:
>  It's possible the text could be cropped and contextual meaning of the image would be lost. I think we may need to go back to having a user-specified size for the image and always showing the entire image at that size.

@donnapep 
If we want to provide users some flexibility, there are two ways we can do it while maintaining the visual integrity of the grid:
- Stick to fixed image height, but allow them to toggle between 'Fill' and 'Fit' scaling. See how Figma does it below:
https://user-images.githubusercontent.com/72030923/125239911-f97d9d00-e306-11eb-9e6c-09ba690415aa.mov

- Allow users to adjust the image size, but for the entire grid so that all cards will have the same image aspect ratio. 

I'm still not sure this level of flexibility is that crucial for users. In most cases the image is just a visual aid to give a glimpse of what the course is about. I feel like we can go ahead with the fixed image height for now, and see if we receive any feedback on that.